### PR TITLE
Moving and Jumping with Left hand

### PR DESCRIPTION
**Pull request by cesarrodrig**

### CODE REVIEW

File: MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java [hunk 1]
```diff
@@ -3,25 +3,35 @@
...
 
+	public enum HandType {
+		RIGHT,
+		LEFT,
+	};
+
 	private static volatile LeapMotionMouse instance = null;
 
 	private Minecraft mc = FMLClientHandler.instance().getClient();
+	private GameSettings settings = Minecraft.getMinecraft().gameSettings;
```
Comment (Nautigsam) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java]: Could you modify this to "mc.gameSettings" please?

Comment (Nautigsam) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java]: or you can just do mc.gameSettings actually.

File: MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java [hunk 2]
```diff
@@ -230,31 +329,111 @@ private boolean pollRightFist() {
...
+
+		// 'debouncing' the fist and flat hand gestures
+		if (grabStrength + LEFT_GRAB_TOLERANCE >= 1.0f) {
+			if (fistLeftHandTicks < positive) fistLeftHandTicks += 1; // prevent overflowing
 		} else {
 			changeOccurred = true;
-			flatHandTicks = 0;
-			rightButtonUp();
+			fistLeftHandTicks = 0;
+			// TODO: what here?
```
Comment (Nautigsam) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java:370]: what here? ;)

File: MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java [hunk 3]
```diff
@@ -230,31 +329,111 @@ private boolean pollRightFist() {
...
+			flatLeftHandTicks = 0;
+			stopJumping();
 		}
 
-		if (fistHandTicks >= positive) {
+		if (fistLeftHandTicks >= positive) {
 			changeOccurred = true;
-			leftButtonDown();
-		} else if (flatHandTicks >= positive) {
+			// TODO: what here?
```
Comment (Nautigsam) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java:382]: what here? ;)

File: MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java (hunk 2 above)
Comment (cesarrodrig) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java:370]: I was thinking we could open the inventory menu, but I don't know.

File: MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java (hunk 1 above)
Comment (cesarrodrig) [inline MineLeapMod/src/com/nautigsam/mineleapmod/LeapMotionMouse.java]: yeap

**Pull request merged**